integration-tests/scripts/spi

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# Copyright (C) 2015 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Script to run the integration tests in the Snappy Product Integration.
+
+set -x
+
+export GOPATH="$(mktemp -d)"
+trap "rm -rf $GOPATH"EXIT
+export PATH="$PATH:$GOPATH/bin"
```

Comment:
Do you need to build anything that might be on the system already (snappy/snapd)? Perhaps $GOPATH/bin:$PATH would be better so that you always get the locally built versions?